charts/areaCharts/StackedAreaTransform.ts

```diff
@@ -163,6 +165,42 @@ export class StackedAreaTransform extends ChartTransform {
         return groupedData
     }
 
+    @computed get xAxis() {
+        const { xDomainDefault } = this
+        const chart = this.chart
+        const axis = chart.xAxisOptions.toHorizontalAxis()
+        axis.updateDomainPreservingUserSettings(xDomainDefault)
+        axis.tickFormat = chart.formatYearFunction as any
+        axis.hideFractionalTicks = true
+        axis.hideGridlines = true
+        return axis
+    }
+
+    @computed private get yDomainDefault(): [number, number] {
+        const yValues = this.allStackedValues.map(d => d.y)
+        return [0, max(yValues) ?? 100]
+    }
+
+    @computed get yAxis() {
+        const { isRelativeMode, yDimensionFirst } = this
+        const { chart, yDomainDefault } = this
+
+        const axis = chart.yAxisOptions.toVerticalAxis()
+        if (isRelativeMode) axis.domain = [0, 100]
```

Comment:
This is the actual fix. If is relative mode we always want to set the domain to [0, 100], even if there were other hard codes mins and maxes.